src/groups/bmq/bmqc/bmqc_orderedhashmapwithhistory.h

```diff
@@ -655,21 +659,17 @@ OrderedHashMapWithHistory<KEY, VALUE, HASH, VALUE_TYPE>::insert(
 }
 
 template <class KEY, class VALUE, class HASH, class VALUE_TYPE>
-inline bool
+inline int
 OrderedHashMapWithHistory<KEY, VALUE, HASH, VALUE_TYPE>::gc(TimeType now,
                                                             unsigned batchSize)
 {
     // Try to advance to either a young item, or to the end of the batch, or to
     // the end of collection.
-    // Return 'true' if at the end of the batch to be called as soon as
-    // possible.  Otherwise, return 'false' to be called at the next timeout.
-    // In either case, resume iteration from where we stopped (or wrap around).
-    // 'erase' can set the iterator back to erase item if its expiration time
-    // is sooner than the current one.
```

Comment:
This comment is not easy to understand, and I replaced it with return values description in declaration. Also, for better comments we should update & merge this
https://github.com/bloomberg/blazingmq/pull/498